lib/routes/chikubi/index.ts

```diff
@@ -0,0 +1,75 @@
+import { Route, Data } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { processItems } from './utils';
+
+export const route: Route = {
+    path: '/:category?',
+    categories: ['other'],
+    example: '/chikubi',
+    parameters: { category: '分類，見下表，默認爲最新' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: 'Category',
+    maintainers: ['snowagar25'],
+    handler,
+    description: `| 最新 | 殿堂 | 動畫 | VR | 漫畫 | 音聲 | CG |
+  | ------ | ---- | ----- | -- | ----- | ----- | -- |
+  | (empty) | best | video | vr | comic | voice | cg |`,
+    radar: [
+        {
+            source: ['chikubi.jp/:category', 'chikubi.jp/'],
+            target: '/:category',
+        },
+    ],
+};
+
+const categoryMap = {
+    '': { url: '/page/1', title: '最新', selector: '.article_list_area > article > a' },
+    best: { url: '/best-nipple-article', title: '殿堂', selector: '.article-list:first .title > a' },
+    video: { url: '/nipple-video', title: '動畫', selector: 'ul.video_list > li > a' },
+    vr: { url: '/nipple-video-category/cat-nipple-video-vr', title: 'VR', selector: 'ul.video_list > li > a' },
+    comic: { url: '/comic', title: '漫畫', selector: '.section:nth-of-type(2) .list-doujin .photo a' },
+    voice: { url: '/voice', title: '音聲', selector: 'ul.list-doujin > li > .photo > a' },
+    cg: { url: '/cg', title: 'CG', selector: 'ul.list-doujin > li > .photo > a' },
+};
+
+async function handler(ctx): Promise<Data> {
+    const category = ctx.req.param('category') ?? '';
+    const baseUrl = 'https://chikubi.jp';
+
+    const { url, title, selector } = categoryMap[category];
+
+    const response = await got(`${baseUrl}${url}`);
+    const $ = load(response.data);
+
+    let list = $(selector)
+        .toArray()
+        .map((item) => {
+            const $item = $(item);
+            return {
+                title: $item.text().trim(),
+                link: new URL($item.attr('href') ?? '', baseUrl).href,
+            };
+        });
+
+    // 限制殿堂最多獲取30個
+    if (category === 'best') {
+        list = list.slice(0, 30);
+    }
+
+    // 獲取內文
+    const items = await processItems(list);
```

Comment:
Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

For the latest post, you can use the endpoint of `/wp-json/wp/v2/posts`

For categories, you can find the category id first using `/wp-json/wp/v2/categories` with the search parameter `slug`. The id for `vr` is `486`. Then you can list recent posts with the required tag using `/wp-json/wp/v2/posts` with the search parameter `categories`, which will be `486` for `vr`. If you want categories and tags for each post in the posts API, you can add the `_embed` search parameter.

To adjust the no. of articles returned to the posts API, you can pass the `per_page` search parameter.

Reply:
After conducting some research, I found that the website author has divided the content into two types: "posts" and "works."

While using `/wp-json/wp/v2/posts` to retrieve the latest post is correct, but when I try with `/wp-json/wp/v2/posts?categories=486`, I only get 8 results from `https://chikubi.jp/category/vr`, which isn't the `https://chikubi.jp/nipple-video-category/cat-nipple-video-vr` that I'm looking for.

I'm unsure if I missed something or misunderstood, but I can't seem to find an API to retrieve the "works" content. And the only section that requires adjusting the quantity, "best", doesn't appear in `wp-json/wp/v2/categories`. 

So, I'd prefer not to use the API just for fetching the latest post as it would become the only exception.